packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1039,7 +1042,9 @@ export const createAzureCommunicationCallWithChatAdapter: ({ userId, displayName
 export const _createAzureCommunicationCallWithChatAdapterFromAdapters: (callAdapter: CallAdapter, chatAdapter: ChatAdapter) => CallWithChatAdapter;
 
 // @public
-export const createAzureCommunicationCallWithChatAdapterFromClients: ({ callClient, callAgent, callLocator, chatClient, chatThreadClient }: AzureCommunicationCallWithChatAdapterFromClientArgs) => Promise<CallWithChatAdapter>;
+export function createAzureCommunicationCallWithChatAdapterFromClients(callClient: StatefulCallClient, callAgent: CallAgent, callLocator: CallAdapterLocator | TeamsMeetingLinkLocator, chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient, options?: {
```

Comment:
Should we add this option to createAzureCommunicationCallWithChatAdapter as well? We should make last param as CallWithChatAdapterOptions instead

Then declare
type CallWithChatAdapterOption = CallAdapterOption && ChatAdapterOption

Reply:
How does inline image previously work in our callWithChat sample app?

Reply:
We only caught this issue when we just updated selfhost app yesterday. So it technically never worked / tested until today. 
We can for sure create a `type CallWithChatAdapterOption = CallAdapterOption && ChatAdapterOption` instead it might make the implementation a little cleaner. 

We shouldn't need to include it in `createAzureCommunicationCallWithChatAdapter` cause when we user that factory method the chat adapter grabs the credential when it creates the chat client itself and sends it down stream .

Reply:
Ohh right! Cause we pass in the token and create credential there